### ISSUE TITLE
Does not go full screen on KDE Plasma desktop (Which is also used on the Steam Deck as its desktop)

### ISSUE THREAD

**Issue by linlinxza** [Legend-of-Dragoon-Modding/Severed-Chains#1129]
![Does not go fullscreen on KDE Plasma](https://github.com/Legend-of-Dragoon-Modding/Severed-Chains/assets/11242593/e14d7af2-d8df-4cd8-836c-c9e2f7901ec2)

As of commit https://github.com/Legend-of-Dragoon-Modding/Severed-Chains/commit/f4885076eb9269d5b856792969acb53afea33bee

**Comment by LordMonoxide** [comment]
- Looks like the window was resized down so that it didn't overlap the taskbar
- Disabling title bar and frame in the window options makes it work as expected
- Gonna have to look into glfw's window options and see if they have any documentation on interacting with various Linux window managers

**Comment by linlinxza** [comment]
Yeah. It's just the lower panel that that window needs to be able to overlap it. I think once this is fixed, it will work on any desktop just fine.

**Comment by B-Nerd** [comment]
still not fixed 

<img width="1928" height="1080" alt="Image" src="https://github.com/user-attachments/assets/26897013-5e53-44b9-a2da-3c684b98e01f" />
<img width="1918" height="1080" alt="Image" src="https://github.com/user-attachments/assets/34c735ce-ac6f-4faa-8d88-c9ad5aa3b78a" />
